README.md

```diff
@@ -60,18 +61,22 @@ Or, passing parameter values:
       - { role: cmacrae.sensu, sensu_master: true, sensu_include_dashboard: true  }
 ```
 
+## Ansible version support
+All changes to this role are actively tested with the last two stable versions of Ansible to ensure compatibility. As such, this role
+only officially supports running with the last two stable releases of Ansible, which aligns with the [Ansible support model](http://docs.ansible.com/ansible/latest/release_and_maintenance.html#release-status).
```

Comment:
So I see the min version as 2.3 in the meta, how are you testing multiple versions? Sorry I am still pretty unfamiliar with ansible.

Reply:
I'm still working on getting official 2.3 testing in Travis. Currently, I am using what Jeff Geerling has in his docker images which is 2.4. I'm hoping to get a solid way to define the Ansible versions along with the OS's. Currently, I am just doing syntax checks locally with 2.3 to make sure (notably there's only 1 large new feature in 2.4 that breaks 2.3 entirely). 

tl;dr - Manually at this point in time, but not for long as I rework the integration tests. That said, this stable is still true, I'm actively ensuring 2.3 support is maintained and will handle any issue with it breaking.